apps/music/js/app.js

```diff
@@ -175,8 +175,11 @@ viewStack.addEventListener('change', (evt) => {
 
   document.body.dataset.activeViewUrl = viewUrl;
 
-  playerButton.hidden = viewUrl === VIEWS.PLAYER.URL || !isPlaying;
-  activityDoneButton.hidden = viewUrl !== VIEWS.PLAYER.URL || !activity;
+  var showingPlayer = viewUrl === VIEWS.PLAYER.URL;
+  playerButton.hidden = showingPlayer || !isPlaying;
+  activityDoneButton.hidden = !showingPlayer || !activity;
+  client.method('enableNFC', showingPlayer);
```

Comment:
Both app.js and endpoint.js are loaded in the same context. You can just directly call `enableNFC()` from here instead of going across the bridge.